.github/azure-pipelines/templates/build.yaml

```diff
@@ -16,7 +16,7 @@ steps:
     - script: 'npm run build'
       displayName: 'Build: wiremock-captain'
 
-    - script: 'npm run test'
+    - script: 'npm run coverage'
```

Comment:
shouldn't test and coverage be different steps with the test preceding coverage?

Reply:
`coverage` is a superset of test (it just tests with --coverage enabled).  There's not really a reason to run tests _without_ coverage and then run them again _with_ coverage (unless your line numbers for your unit tests get all messed up, but with jest coverage that usually isn't an issue).